spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -31,47 +31,117 @@ object SpdxLicenseAliasMapping {
      */
     private val customNames = mapOf(
         "AFLv2.1" to AFL_2_1,
+        "afl" to AFL_2_1,
+        "afl-2" to AFL_2_0,
+        "afl2" to AFL_2_0,
+        "afl2.0" to AFL_2_0,
+        "afl2.1" to AFL_2_1,
+        "agpl" to AGPL_3_0_ONLY,
         "ALv2" to APACHE_2_0,
         "APL2" to APACHE_2_0,
         "APLv2.0" to APACHE_2_0,
         "ASL" to APACHE_2_0,
         "Apache" to APACHE_2_0,
         "Apache-2" to APACHE_2_0,
         "Apache2" to APACHE_2_0,
+        "alv2" to APACHE_2_0,
+        "apache" to APACHE_2_0,
+        "apache-license" to APACHE_2_0,
+        "apache_2_0" to APACHE_2_0,
+        "apache_v2" to APACHE_2_0,
+        "artistic_1" to ARTISTIC_1_0,
+        "artistic_2" to ARTISTIC_2_0,
+        "asl" to APACHE_2_0,
         "BOOST" to BSL_1_0,
-        "BSD" to BSD_2_CLAUSE,
+        "boost" to BSL_1_0,
+        "Bouncy" to MIT,
+        "bouncy" to MIT,
+        "bouncy-license" to MIT,
+        "BSD" to BSD_3_CLAUSE,
         "BSD-3" to BSD_3_CLAUSE,
+        "BSD*" to BSD_3_CLAUSE,
         "BSD-Style" to BSD_3_CLAUSE,
         "BSD-like" to BSD_3_CLAUSE,
         "BSD-style" to BSD_3_CLAUSE,
         "BSD2" to BSD_2_CLAUSE,
         "BSD3" to BSD_3_CLAUSE,
+        "bsd" to BSD_3_CLAUSE,
+        "bsd-license" to BSD_3_CLAUSE,
+        "bsd-licensed" to BSD_3_CLAUSE,
+        "bsd-like" to BSD_3_CLAUSE,
+        "bsd-style" to BSD_3_CLAUSE,
+        "bsd2" to BSD_2_CLAUSE,
+        "bsd3" to BSD_3_CLAUSE,
+        "bsl" to BSL_1_0,
+        "bsl1.0" to BSL_1_0,
         "CC0" to CC0_1_0,
+        "cc0" to CC0_1_0,
         "CDDL" to CDDL_1_0,
+        "cddl" to CDDL_1_0,
+        "cddl1.0" to CDDL_1_0,
+        "cddl1.1" to CDDL_1_1,
         "CPL" to CPL_1_0,
         "EDL-1.0" to BSD_3_CLAUSE,
+        "efl" to EFL_2_0,
         "EPL" to EPL_1_0,
+        "epl" to EPL_1_0,
+        "epl1.0" to EPL_1_0,
+        "epl2.0" to EPL_2_0,
+        "eupl" to EUPL_1_0,
+        "eupl1.0" to EUPL_1_0,
+        "eupl2.0" to EUPL_2_0,
```

Comment:
@tsteenbe, there actually is no EUPL_2_0 in SPDX. Did you mean to map `"eupl1.2" to EUPL_1_2`?

Reply:
Yes, should have been EUPL_1_2